app-modules/service-management/src/Filament/Resources/ServiceRequestTypeResource/Pages/ServiceRequestTypeEmailTemplatePage.php

```diff
@@ -67,6 +73,23 @@ public function getRelationManagers(): array
 
     public function form(Form $form): Form
     {
+        if (ServiceRequestTypeEmailTemplateTabs::active()) {
+            return $form
+                ->schema([
+                    Tabs::make('EmailTemplateRoles')
+                        ->statePath('activeEmailTemplateRole')
+                        ->persistTab()
+                        ->id('email-template-role-tabs')
+                        ->tabs(array_map(
+                            fn (ServiceRequestTypeEmailTemplateRole $role) => Tab::make($role->getLabel())
+                                ->schema($this->emailTemplateSchema($role->value))
```

Comment:
I would probably call this method `getEmailTemplateFormSchemaForRole()` for clarity